selfdrive/car/ford/interface.py

```diff
@@ -69,32 +59,22 @@ def get_params(candidate, fingerprint, vin=""):
     ret.lateralTuning.pid.kf = 1. / MAX_ANGLE   # MAX Steer angle to normalize FF
     ret.steerActuatorDelay = 0.1  # Default delay, not measured yet
     ret.steerRateCost = 1.0
-
-    f = 1.2
-    tireStiffnessFront_civic *= f
-    tireStiffnessRear_civic *= f
-
     ret.centerToFront = ret.wheelbase * 0.44
-
+    tire_stiffness_factor = 0.444  # not optimized
```

Comment:
this should be 0.6, right?

Reply:
yeah 20% higher. wanted to double check on whether this number was based on a meaningful method, so I defaulted it at the time

Reply:
subjective tuning